test/test_coretasks.py

```diff
@@ -0,0 +1,119 @@
+# coding=utf-8
+"""coretasks.py tests"""
+from __future__ import unicode_literals, absolute_import, print_function, division
+
+import pytest
+
+from sopel import coretasks
+from sopel.module import VOICE, HALFOP, OP, ADMIN, OWNER
+from sopel.tools import Identifier
+from sopel.test_tools import MockSopel, MockSopelWrapper
+from sopel.trigger import PreTrigger, Trigger
+
+
+@pytest.fixture
+def sopel():
+    bot = MockSopel("Sopel")
+    return bot
+
+
+@pytest.fixture
+def sopel_bot(sopel):
+    pretrigger = PreTrigger(Identifier("Foo"), "PING abc")
+    bot = MockSopelWrapper(sopel, pretrigger)
+    bot.privileges = dict()
+    bot.users = dict()
```

Comment:
Both `bot.privileges` and `bot.users` can be assigned their default value in the `sopel` fixture above.

I'd even argue that this fixture can be avoided, since tests below create their own `PreTrigger`, so you can use this instead:

    sopel_bot = MockSopelWrapper(sopel, trigger)

I added the suggestions on one test as an example.